core/app/modules/import-export/processes/revert.php

```diff
@@ -0,0 +1,166 @@
+<?php
+namespace Elementor\Core\App\Modules\ImportExport\Processes;
+
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Elementor_Content;
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Revert_Runner_Base;
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Plugins;
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Site_Settings;
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Taxonomies;
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Templates;
+use Elementor\Core\App\Modules\ImportExport\Runners\Revert\Wp_Content;
+use Elementor\Core\App\Modules\ImportExport\Module;
+use Elementor\Core\App\Modules\ImportExport\Runners\Runner_Interface;
+
+class Revert {
+
+	/**
+	 * @var Revert_Runner_Base[]
+	 */
+	protected $runners = [];
+
+	private $import_sessions;
+
+	private $revert_sessions;
+
+	/**
+	 * @throws \Exception
+	 */
+	public function __construct() {
```

Comment:
I prefer to force the developer to intentionally register the default runners, that way is easier to prevent them from registering the runners without any reason (the runners are only needed when we run the 'run' function.)